chef/cookbooks/crowbar/recipes/default.rb

```diff
@@ -390,7 +390,7 @@
     realm: realm
   )
 
-  notifies :reload, resources(service: "apache2")
+  notifies :restart, resources(service: "apache2"), :immediately
```

Comment:
@vuntz We could also just add the needed modules to https://github.com/crowbar/crowbar-core/blob/master/chef/cookbooks/apache2/recipes/default.rb#L205 (there's already a comment for suse). Downside is that we than have the proxy_* modules (and all others needed by crowbar) also for the dashboard.